cmd/insprd/operators/kafka/nodes/converter.go

```diff
@@ -191,6 +218,9 @@ func toNode(kdep *kubeApp.Deployment) (meta.Node, error) {
 	if err != nil {
 		return meta.Node{}, err
 	}
+	if len(kdep.Spec.Template.Spec.Containers) == 0 {
+		return meta.Node{}, ierrors.NewError().Message("node does not contain a container").InvalidApp().Build()
```

Comment:
Maybe make it so the expected return is *meta.Node, so this would return (nil, error)